app/views/idv/doc_auth/ssn.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if flow_session['pii_from_doc']['ssn'] %>
   <%= render 'idv/doc_auth/ssn_update', flow_session: flow_session %>
 <% else %>
-  <%= render 'idv/doc_auth/ssn_init', flow_session: flow_session %>
+  <%= render 'idv/doc_auth/ssn_init', flow_session: flow_session, should_render_id_success_alert: true %>
```

Comment:
I forget, can we pass locals with a `?` at the end? they are valid symbols but I forget if actionview/ERB handles them. If we can, let's rename to a simpler predicate?

```suggestion
  <%= render 'idv/doc_auth/ssn_init', flow_session: flow_session, render_id_success_alert?: true %>
```

Reply:
Unfortunately it doesn't seem to support question mark symbols, I get an undefined method error when I try doing this substitution (in all the files, not just this one). I can do `render_id_success_alert` if you'd like but it could be confused for a method that's actually doing the rendering instead of just a boolean value